src/login/components/LoginApp/ResetPassword/ResetPasswordMain.js

```diff
@@ -1,18 +1,17 @@
 import React, { useEffect } from "react";
 import { withRouter } from "react-router-dom";
 import i18n from "../../../translation/InjectIntl_HOC_factory";
-import { useSelector, useDispatch } from 'react-redux';
+import { useDispatch } from 'react-redux';
 import { getResetPassword } from "../../../redux/actions/getResetPasswordActions"
 import ResetPasswordForm from "./ResetPasswordForm";
 import EmailLinkSent from "./EmailLinkSent";
 
 const ResetPasswordMain = (props) => {
   const dispatch = useDispatch();
-  const csrf_token = useSelector(state => state.resetPassword.csrf_token);
 
   useEffect(() => {
     dispatch(getResetPassword());
-  }, [csrf_token]);
+  }, []);
```

Comment:
Maybe there is an unexpected re-render of this component, because I can still see two GET_RESET_PASSWORD and GET_RESET_PASSWORD_SUCCESS.

in redux:
![Screenshot 2021-06-03 at 15 45 55](https://user-images.githubusercontent.com/30963614/120655461-fdeea480-c482-11eb-9479-5466dc89165a.png)

in network tab:
![Screenshot 2021-06-03 at 15 37 59](https://user-images.githubusercontent.com/30963614/120655723-3f7f4f80-c483-11eb-9211-1f0b981fe993.png)
![Screenshot 2021-06-03 at 15 38 18](https://user-images.githubusercontent.com/30963614/120655676-34c4ba80-c483-11eb-856d-7afe9db76eb6.png)